.eslintrc.js

```diff
@@ -1,107 +1,81 @@
 module.exports = {
-  env: { // Define the environment where the linting will take place
-    browser: true,
-    es6: true,
-    node: true,
-  },
-  plugins: [
-    'import',
-    'jsdoc',
-    '@nx',
-  ],
-  extends: [
-    'airbnb-base', // We extend our linting to follow the airbnb rules
-    'plugin:import/recommended',
-  ],
-  parser: '@babel/eslint-parser', // Parser that'll help linting react demos
+  plugins: ["import", "jsdoc", "@nx", "prettier"],
```

Comment:
Maybe a brief explanation on why are we using those for future references

Reply:
As we discussed Jose and i, we decided to mark it as an improvement for a next task.